README.md

```diff
@@ -1,32 +1,169 @@
 # Prompt flow
 
+[![banner](examples/tutorials/quick-start/media/PF_banner.png)](https://microsoft.github.io/promptflow)
+
 [![Python package](https://img.shields.io/pypi/v/promptflow)](https://pypi.org/project/promptflow/)
 [![PyPI - Downloads](https://img.shields.io/pypi/dm/promptflow)](https://pypi.org/project/promptflow/)
 [![License: MIT](https://img.shields.io/github/license/microsoft/promptflow)](https://github.com/microsoft/promptflow/blob/main/LICENSE)
+[![](https://dcbadge.vercel.app/api/server/bnXr6kxs?compact=true&style=flat)](https://discord.gg/bnXr6kxs)
+
+> Welcome to join us to make Prompt flow!
 
-> Welcome to join us to make Prompt flow better by
-> participating [discussions](https://github.com/microsoft/promptflow/discussions),
-> opening [issues](https://github.com/microsoft/promptflow/issues/new/choose),
-> submitting [PRs](https://github.com/microsoft/promptflow/pulls).
+[Documentacion](https://microsoft.github.io/promptflow/) • [Issues](https://github.com/microsoft/promptflow/issues/new/choose)  • [Discord](https://discord.gg/bnXr6kxs) •  [Discussions](https://github.com/microsoft/promptflow/discussions) • [Contribute PRs](https://github.com/microsoft/promptflow/pulls).
```

Comment:
Maybe we could combine this with the icons we have above 
Something like 
![image](https://github.com/microsoft/promptflow/assets/47586720/c840b2e6-5a5e-4664-b277-cb6ccbc80309)